modules/autovoice.cpp

```diff
@@ -335,16 +395,16 @@ class CAutoVoiceMod : public CModule {
         PutModule(t_f("User {1} removed")(sUser));
     }
 
-    CAutoVoiceUser* AddUser(const CString& sUser, const CString& sHost,
+    CAutoVoiceUser* AddUser(const CString& sUser, const CString& sHosts,
                             const CString& sChans) {
         if (m_msUsers.find(sUser) != m_msUsers.end()) {
             PutModule(t_s("That user already exists"));
             return nullptr;
         }
 
-        CAutoVoiceUser* pUser = new CAutoVoiceUser(sUser, sHost, sChans);
+        CAutoVoiceUser* pUser = new CAutoVoiceUser(sUser, sHosts, sChans);
         m_msUsers[sUser.AsLower()] = pUser;
-        PutModule(t_f("User {1} added with hostmask {2}")(sUser, sHost));
+        PutModule(t_f("User {1} added with hostmask(s) {2}")(sUser, sHosts));
```

Comment:
https://github.com/znc/znc/blob/master/modules/autoop.cpp#L490 Is that also wrong?